bucket/wsltty.json

```diff
@@ -10,12 +10,10 @@
     "autoupdate": {
         "url": "https://github.com/mintty/wsltty/releases/download/$version/wsltty-$version-install.exe#/dl.7z"
     },
+    "pre_install": "(\"@cd %~dp0\", (Get-Content \"$dir\\install.bat\")) | Out-File -Encoding ASCII \"$dir\\install.bat\"",
```

Comment:
This will not keep @echo off at first, so it should print command in terminal. Also no popd so users context will be changed

Reply:
I think you may find those concerns are satisfied. The "@" before "cd" keeps echo off at first. The user's context is not changed because program "install.bat" runs in its own process as usual.